README.md

```diff
@@ -1,9 +1,7 @@
 # Mountpoint for Amazon S3 CSI Driver
 
 ## Overview
-The Mountpoint for Amazon S3 Container Storage Interface (CSI) Driver allows your Kubernetes applications to access Amazon S3 objects through a file system interface. Built on [Mountpoint for Amazon S3](https://github.com/awslabs/mountpoint-s3), the Mountpoint CSI driver presents an Amazon S3 bucket as a storage volume accessible by containers in your Kubernetes cluster. The Mountpoint CSI driver implements the [CSI](https://github.com/container-storage-interface/spec/blob/master/spec.md) specification for container orchestrators (CO) to manage storage volumes.
-
-For Amazon EKS clusters, the Mountpoint for Amazon S3 CSI driver is also available as an [EKS add-on](https://docs.aws.amazon.com/eks/latest/userguide/eks-add-ons.html) to provide automatic installation and management.
+The Mountpoint for Scality S3 Container Storage Interface (CSI) Driver allows your Kubernetes applications to access Scality S3 objects through a file system interface. Built on [Mountpoint for Amazon S3](https://github.com/awslabs/mountpoint-s3), the Mountpoint CSI driver presents an Scality S3 bucket as a storage volume accessible by containers in your Kubernetes cluster. The Mountpoint CSI driver implements the [CSI](https://github.com/container-storage-interface/spec/blob/master/spec.md) specification for container orchestrators (CO) to manage storage volumes.
```

Comment:
```suggestion
The Mountpoint for Scality S3 Container Storage Interface (CSI) Driver allows your Kubernetes applications to access Scality S3 objects through a file system interface. Built on [Mountpoint for Amazon S3](https://github.com/awslabs/mountpoint-s3), the Mountpoint CSI driver presents a Scality S3 bucket as a storage volume accessible by containers in your Kubernetes cluster. The Mountpoint CSI driver implements the [CSI](https://github.com/container-storage-interface/spec/blob/master/spec.md) specification for container orchestrators (CO) to manage storage volumes.
```

___
There's a line in `# Features` section as well.
___
Can we have a detail on why we need a fork a what changes

Reply:
yes we will have a docs PR in this ticket. https://scality.atlassian.net/browse/S3CSI-17

What do you mean by "why we need a fork a what changes"

Reply:
I mean what did we change compared to the amazon's implementation.

Why don't we go with the amazon version directly ?